src/workflow/reset.c

```diff
@@ -14,7 +14,6 @@
 
 #include "reset.h"
 #include "confirm.h"
-#include "password.h"
 #include "workflow.h"
```

Comment:
Builds also without including workflow.h. I think none of that is used here in reset anymore.

Reply:
done